src/main/java/at/hannibal2/skyhanni/features/fishing/SeaCreatureFeatures.kt

```diff
@@ -37,6 +37,7 @@ object SeaCreatureFeatures {
     private var lastRareCatch = SimpleTimeMark.farPast()
     private val rareSeaCreatures = TimeLimitedSet<Mob>(6.minutes)
     private val entityIds = TimeLimitedSet<Int>(6.minutes)
+    var lastRareSeaCreatureDespawn = SimpleTimeMark.farPast()
```

Comment:
the last time alone is no good check. rather check the location of where the mob died, and then use the distance to player for allowing to check if the player picked it up. also should check if the mob type matches the drop, optionally.